xgboost_ames_housing/demo/mockup-data-xgboost-build-train-deploy.ipynb

```diff
@@ -0,0 +1,1015 @@
+{
```

Comment:
What is the difference between this file and `ames-xgboost-build-train-deploy.ipynb`?